contracts/storage/ProfileStorage.sol

```diff
@@ -93,6 +90,14 @@ contract ProfileStorage {
         emit AskUpdated(identityId, ask);
     }
 
+    function getReward(uint72 identityId) external view returns (uint96) {
+        return profiles[identityId].reward;
```

Comment:
general comment on wording - "reward" sounds problematic.
We should stick with "fees" (accumulated fees, op fees etc) for node operator fees, and have "reward" be used for the rest (tokens accumulating in stake)

Reply:
also makes reviewing the code harder

Reply:
Changed